server/config/config.go

```diff
@@ -100,7 +100,7 @@ func readConfigFile(vp *viper.Viper) error {
 	}
 
 	_, fileNotFound := err.(viper.ConfigFileNotFoundError)
-	if fileNotFound {
+	if fileNotFound || vp.GetString("config") == "" {
```

Comment:
There is a bug with the current implementation that if you don't provide a `config.yaml` file the server fails @schoren I'm going to create an issue in our board with the specs

Reply:
Bug ticket https://github.com/kubeshop/tracetest/issues/2071